src/components/countrycomponent.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/prop-types */
+import { Link } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import style from './countrycomponent.module.css';
```

Comment:
- [ ] Please remove this eslint disabled rule and write proptypes for the component.